eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/session/retry/EventMeshTcpRetryer.java

```diff
@@ -37,9 +37,9 @@ public class EventMeshTcpRetryer {
 
     private EventMeshTCPServer eventMeshTCPServer;
 
-    private DelayQueue<RetryContext> retrys = new DelayQueue<RetryContext>();
+    private final DelayQueue<RetryContext> retrys = new DelayQueue<RetryContext>();
```

Comment:
How about change like this
```java
private final DelayQueue<RetryContext> retrys = new DelayQueue<>();
```

Reply:
yeah ofcourse it seems better. That is how we always declare Map<> object. Sorry I could not notice this.

Reply:
@mxsm I have updated the pull request. Please review.